scrypt.js

```diff
@@ -2,4 +2,5 @@ document.body.innerHTML = '<h1>hello js</h1>';
 console.log('test1')
 console.log('test2!!!!!!!!!!')
 console.log('test2!!!!!!!!!!')
-console.log('test3')
\ No newline at end of file
+console.log('test3')
+console.log('test45')
```

Comment:
It is better to add an empty line to the end of the file. You can see that github also suggests it
<img width="251" alt="image" src="https://user-images.githubusercontent.com/16056824/188245030-7e54eaea-62b4-40f8-8482-9a3ee7aaac13.png">